<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stockCriteriaEditor</class>
 <widget class="QWidget" name="stockCriteriaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assets Categories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Critérios para compra de ações</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Indicadores:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_indicators">
       <property name="placeholderText">
        <string>Escrever quais indicadores vou analisar para comprar uma ação. Colocar faixar de valores</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Análise do investidor:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_analysis">
       <property name="placeholderText">
        <string>Escrever o que devo avaliar ao buscar uma empresa. Como ela ganha dinheiro? Quais são os riscos do negócio?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Como fazer Valuation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_valuation">
       <property name="placeholderText">
        <string>Descrever passo a passo da metodologia que utilizarei para fazer o valuation</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Demais critérios:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_more_criteria">
       <property name="placeholderText">
        <string>Escrever critérios adicionais que eu gostaria de avaliar. Categorias de Lynch? Dimensões de Fisher?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Número ideal de ações e setores desejados:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_number">
       <property name="placeholderText">
        <string>Escrever número mínimo e máximo de ações que eu deveria ter. Quais setores da economia eu quero ter? Quantas empresas em cada setor?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
